### DISCUSSION TITLE
Help me !

### DISCUSSION THREAD

**Discussion by YTLoudYN** [beeware/toga#1904] [Q&A] [answered]
![x1_20230426190906](https://user-images.githubusercontent.com/131869902/234557831-8d3e9d2d-b997-44c8-a629-eed896fcc053.png)
![联想截图_20230426190936](https://user-images.githubusercontent.com/131869902/234557996-32ac1cb4-66d5-4ffe-b02e-86aa3e1408b3.png)
Why not is  path

**Comment by freakboy3742** [comment, marked as answer]
Why would it be a bool? You're opening a file, so the thing you want to get back is a File (or some sort of file-related object).

In this case, you also need to `await` the dialog - `await window.open_file_dialog(...)` to get the file; you'll receive a Pathlib object pointing to the selected file.

If you want an example of using a file dialog, see the [dialogs](https://github.com/beeware/toga/blob/main/examples/dialogs/dialogs/app.py#L60) example in the Toga codebase.

**Reply by YTLoudYN** [reply]
I got in ,Thanks
